x-pack/plugins/endpoint/public/applications/endpoint/components/management.tsx

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  EuiPage,
+  EuiPageBody,
+  EuiPageContent,
+  EuiPageContentBody,
+  EuiPageContentHeader,
+  EuiPageContentHeaderSection,
+  EuiPageHeader,
+  EuiPageHeaderSection,
+  EuiTitle,
+  EuiBasicTable,
+} from '@elastic/eui';
+import { useEndpointListSelector } from '../store/hooks';
+import {
+  endpointListData,
+  endpointListPageIndex,
+  endpointListPageSize,
+  endpointTotalHits,
+} from '../store/endpoint_list/selectors';
+import { EndpointListAction } from '../store/endpoint_list/action';
+
+export const EndpointList = () => {
+  const dispatch = useDispatch<(a: EndpointListAction) => void>();
+  const endpointListResults = useEndpointListSelector(endpointListData);
+  const pageIndex = useEndpointListSelector(endpointListPageIndex);
+  const pageSize = useEndpointListSelector(endpointListPageSize);
+  const totalItemCount = useEndpointListSelector(endpointTotalHits);
+
+  const paginationSetup = {
+    pageIndex,
+    pageSize,
+    totalItemCount,
+    pageSizeOptions: [10, 20, 50],
+    hidePerPageOptions: false,
+  };
+
+  const onTableChange = ({ page }: { page: { index: number; size: number } }) => {
+    const { index, size } = page;
+    dispatch({
+      type: 'userPaginatedEndpointListTable',
+      payload: { pageIndex: index, pageSize: size },
+    });
+  };
+
+  const columns = [
+    {
+      field: 'host.hostname',
+      name: 'Host',
```

Comment:
we should i18n visible names using the i18n package from kibana. Import `i18n` from `@kbn/i18n`, then use it as:

```javascript
i18n.translate('xpack.endpoint.management.list', {
  defaultMessage: 'Host',
})
```